packages/layout/src/components/cards/components/button.module.scss

```diff
@@ -37,4 +37,9 @@
 	padding: 0 0 4px 2px;
 	width: 100%;
 	height: $space-5;
+
+	@media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
```

Comment:
In the context of this PR I can see this is an IE hack, but a comment explaining that would be useful because without the context of this PR it will be pretty hard to work out why this was done.

Reply:
comments added with a description of the fix